src/server.c

```diff
@@ -6046,6 +6046,7 @@ sds genValkeyInfoString(dict *section_dict, int all_sections, int everything) {
                 "mem_not_counted_for_evict:%zu\r\n", freeMemoryGetNotCountedMemory(),
                 "mem_replication_backlog:%zu\r\n", mh->repl_backlog,
                 "mem_total_replication_buffers:%zu\r\n", server.repl_buffer_mem,
+                "mem_replicas_repl_buffer:%zu\r\n", server.pending_repl_data.mem,
```

Comment:
As an end user, it is really hard to know what this name means without reading the docs. Probably we should add "dual channel" to the name so that it is clear that this is related to that feature. WDYT about:

```suggestion
                "mem_replica_dual_channel_buffer:%zu\r\n", server.pending_repl_data.mem,
```

Reply:
YES, the name is a bit unclear, i struggled with this name for a long time... Sadly, we previously used `replicas_repl_buffer_size` before in other fields.